pjnath/src/pjnath/stun_transaction.c

```diff
@@ -399,6 +415,9 @@ PJ_DEF(pj_status_t) pj_stun_client_tsx_retransmit(pj_stun_client_tsx *tsx,
         return PJ_SUCCESS;
```

Comment:
minor: just remove this (or combine it with the below check) since it's more or less the same.